src/Serilog.Settings.Configuration/Settings/Configuration/LoadedConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+using Serilog.Core;
+
+namespace Serilog.Settings.Configuration;
+
+/// <summary>
+/// Contains information about the loaded configuration.
+/// </summary>
+public class LoadedConfiguration
+{
+    readonly IDictionary<string,LoggingLevelSwitch> _logLevelSwitches;
+
+    internal LoadedConfiguration(IDictionary<string, LoggingLevelSwitch> logLevelSwitches)
+    {
+        _logLevelSwitches = logLevelSwitches;
+    }
+
+    /// <summary>
+    /// The log level switches that were loaded from the <c>LevelSwitches</c> section of the configuration.
+    /// </summary>
+    /// <remarks>The keys of the dictionary are the name of the switches without the leading <c>$</c> character.</remarks>
+    public IReadOnlyDictionary<string, LoggingLevelSwitch> LogLevelSwitches => _logLevelSwitches.ToDictionary(e => e.Key.TrimStart('$'), e => e.Value);
```

Comment:
Property allocates `Dictionary` and N strings each call to it.

Reply:
I don't know if it is worth worrying about the initial dollar sign in the name. In fact, this is part of the switch name. What is the goal in removal of the dollar sign?

Reply:
> In fact, this is part of the switch name.

Well, this is debatable: the `LevelSwitches` section does not need to contain the `$` prefix. As found in the README and the `appsettings.json` file of the `Sample` project.

```json
{
  "Serilog": {
    "Using": [ "Serilog.Sinks.Console" ],
    "LevelSwitches": { "controlSwitch": "Verbose" },
```

> Property allocates `Dictionary` and N strings each call to it.

I wouldn't worry too much about that, one call is made per call to `.ReadFrom.Configuration(configuration)`. So for most apps that would be exactly once at startup I guess.

Also, `IDictionary` does not implement `IReadOnlyDictionary`, _[sigh](https://github.com/dotnet/runtime/issues/31001)_. But we could store `Dictionary<string, LoggingLevelSwitch>` instead of `IDictionary<string, LoggingLevelSwitch>` everywhere if the consensus is that this allocation must really be avoided.

Reply:
> the LevelSwitches section does not need to contain the $ prefix.

Then it makes no sense in cutting off the dollar sign if it is not a special symbol. Why to do this? Just return `_logLevelSwitches` as is.

Reply:
I have addressed this concern in 2ecbe0a32c12c9d75941afcd0a6f47fdfd9bf534.

Reply:
👍  $ should be appended in tests.

Reply:
Of course! Fixed in 3f459e1b353a92b92a36064ac04aa0b8d4dad945.